packages/ui-bindings/Organization/getOrganizationById.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * A method to get a single Organization
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2018-1-26
+ */
+
+import { connect } from "react-redux";
+import { gql, graphql, compose } from "react-apollo";
+
+import { AppState } from "@vflows/store/types.js";
+import { getActiveLoginToken } from "@vflows/store/selectors/auth.js";
+
+import { organizationInterface } from "./organizationInterface";
+
+const query = gql`
+query($token: String, $OrganizationId: Int) {
```

Comment:
$OrganizationId could lead to confusion, since variables are typically made camelCase and interfaces / types / classes are made UpperCase. I'll still approve it if you feel otherwise, but consider making variables camelCase

Reply:
I made this decision to keep form confusing it from props.organazationId. Do you think that is a valid reason to not camelCase it?

Reply:
Personally, I would say since they represent the same information they should have the same name. I'll leave it to your discretion though if you want to change it or not